src/tests/audio/tasks/forced_alignment_test.py

```diff
@@ -0,0 +1,194 @@
+"""Tests for forced alignment functions."""
```

Comment:
can you please exec locally only (and not on github) those unit tests that require downloading and using big models? please, check the other test files